node/node.go

```diff
@@ -687,22 +715,6 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 
 	go n.initManagerConnection(connCtx, ready)
 
-	// this happens only on initial start
-	if ready != nil {
-		go func(ready chan struct{}) {
-			select {
-			case <-ready:
-				addr, err := n.RemoteAPIAddr()
-				if err != nil {
-					log.G(ctx).WithError(err).Errorf("get remote api addr")
-				} else {
-					n.remotes.Observe(api.Peer{NodeID: n.NodeID(), Addr: addr}, remotes.DefaultObservationWeight)
-				}
-			case <-connCtx.Done():
-			}
-		}(ready)
-	}
-
```

Comment:
This change makes connections to the local manager use a unix socket or similar local connection (via `connectionbroker`) instead of connecting over TCP. As a consequence, this code, which finds the manager's IP address and adds it to the `Remotes` list to bootstrap `Remotes`, isn't necessary.